### PR TITLE
Js hw4

### PR DESCRIPTION
**Pull request by emma13z**
image

### CODE REVIEW

**Comment by IvanCeban** [comment]
Please merge your previous PR

File: script.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+let removedNo1 = numbers.pop();
+console.log(numbers);
+
+let removedNo2 = numbers.shift();
+console.log(numbers);
+
+// Ex-3 //
+
+let colors = ["red", "green", "blue", "yellow"];
+let index = colors.indexOf ("green");
```
Comment (IvanCeban) [inline script.js]: don't add spaces between method name and (

**Review by IvanCeban — APPROVED** [review]
just remove the spaces before ( and you're good to go

File: script.js (hunk 1 above)
Comment (emma13z) [reply script.js]: got it :)

**Pull request closed without merging**